packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -168,3 +181,65 @@ test('ICA connection can send msg with proto3', async t => {
     message: 'ABCI code: 5: error handling packet: see events for details',
   });
 });
+
+test('Query connection can be created', async t => {
+  const {
+    runUtils: { EV },
+  } = t.context;
+
+  const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
+
+  const account = await EV(orchestration).createQueryConnection('connection-0');
+  t.log('Query Account', account);
+  t.truthy(account, 'createAccount returns an account');
+  t.truthy(
+    matches(account, M.remotable('QueryConnection')),
+    'QueryConnection is a remotable',
+  );
```

Comment:
What is the role of the party making a connection?

This code suggests it's "As the BLD stakers, we can make query connections."
Is that the expected usage?

I suspect it's "As a contract (developer), I can make query connections."
I suggest expressing that as something like...

```
test('Query connection can be created', async t => {
  const contract1 = async ({ orchestration }) => {
    const account = await EV(orchestration).createQueryConnection('connection-0');
    t.log('Query Account', account);
    t.truthy(account, 'createAccount returns an account');
    t.truthy(
      matches(account, M.remotable('QueryConnection')),
      'QueryConnection is a remotable',
    );
  }

  // core eval context
  {
    const { runUtils } = t.context;
    const { EV } = runUtils;
    const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
    await contract1({ orchestration });
  }
});
```

Reply:
The change in `stakingAccountHolder.js` suggests that the contract isn't expected to get all of `orchestration` but rather only the connection (account? are these synonyms?):

```  // core eval context
  {
    const { runUtils } = t.context;
    const { EV } = runUtils;
    const orchestration = await EV.vat('bootstrap').consumeItem('orchestration');
    const account = await EV(orchestration).createQueryConnection('connection-0');
    t.log('Query Account', account);
    await contract1({ queryConnection: account });
  }
```

Reply:
Updated to use this convention